compiler/rustc_mir_build/src/check_unsafety.rs

```diff
@@ -90,23 +91,50 @@ impl<'tcx> UnsafetyVisitor<'_, 'tcx> {
     }
 
     fn emit_deprecated_safe_fn_call(&self, span: Span, kind: &UnsafeOpKind) -> bool {
+        fn parse_rustc_deprecated_safe_2024_attr(attr: &Attribute) -> Option<Symbol> {
```

Comment:
The function is used once and is not too large, it can be inlined and removed.

Reply:
Would labeled-block-break be able to do that as well?

Reply:
Something like `Iterator::find` would work too.

Reply:
Using `Iterator::find` now. Thanks for the suggestion. :)